templates/sios-datakeeper.template.yaml

```diff
@@ -1024,6 +1024,24 @@ Resources:
           - Name: InstanceId
             Selector: $.Reservations[0].Instances[0].InstanceId
             Type: String
+        - name: updateS3BucketName
+          action: aws:updateVariable
+          inputs:
```

Comment:
Looking at the docs for aws:updateVariable, I don't think this format is correct. According to the docs, "[T]his action updates a value for a runbook variable." Which means the Name param needs to match a runbook variable name, and we use QSS3BucketName (on line 951) not S3BucketName.

Actually, you could get rid of this extra automation step, and instead do this logic near line 2640 and 2643 where we pass the template values into the automation doc execution command.